FAQ: mention invite links (#831)
deltachat/deltachat-pages: en/help.md

- Direct 1:1 chat invitation: 
  Tap the QR Code icon <img style="vertical-align:middle; width:1.8em; margin:1px" src="../assets/help/qr-icon.png" />
  on the Delta Chat app main screen.

>   Show the QR code to the other person for scanning.

Reviewer: i added a commit to avoid this PR getting stale

> If the QR code scanning doesn't work in your situation,
> you can also copy an <https://i.delta.chat> invite link to clipboard
> and share it with the other person through a second channel.

Reviewer: at this position, the paragraphs disturbs a bit the idea of a "QR scan-show checklist" as it is currently.

if ppl go for the alternative, the block "Joiner side:" is not valid, needing more abstraction power.

therefore, i'd first describe "QR scan-show checklist" as it is currently (it is short enough), and then before "Both Inviter and Joiner:" (which is valid again also for invite links) go for invite links.

moreover, i'd leave out the https://i.delta.chat  link as it is not helpful in most cases and mention **Share** as well (it is the primary option on android / ios even)

```suggestion
If the QR code scanning doesn't work in your situation,
you can also **Share** (or **Copy to Clipboard**) an equivalent invite link
to the other person through a second channel.
```

Reviewer: i added a commit to avoid this PR getting stale
